pressbooks-openstax-import.php

```diff
@@ -80,11 +91,59 @@ function poi_add_initialize_import(){
  * @return array
  */
 function poi_import_file_types( $allowed_file_types ) {
+
 	if ( is_array( $allowed_file_types ) &&  array_key_exists( 'pb_import_file_types', $allowed_file_types ) ) {
 		$allowed_file_types['pb_import_file_types']['zip'] = 'application/zip';
 	}
 
 	return $allowed_file_types;
 
 }
+
 add_filter( 'pb_import_file_types', 'poi_import_file_types' );
+
+/**
+ * Verify WP QuickLaTeX is installed and active, notice goes away once activated or dismissed
+ */
+function check_latex() {
+	$path = 'wp-quicklatex/wp-quicklatex.php';
+
+	$all_plugins = get_plugins();
+
+	if ( is_plugin_active_for_network( $path ) ) {
+		// quickLaTeX plugin is installed and active, do nothing
+	} else if ( isset( $all_plugins[ $path ] ) ) {
+		// quickLaTex is installed but not active, remind the network administrator to activate it
+		add_action( 'network_admin_notices', function () {
+			// don't annoy them anymore if they've dismissed the activate notice
+			if ( class_exists( 'PAnD' ) && ! PAnD::is_admin_notice_active( 'activate-notice-forever' ) ) {
+				return;
+			}
+			// annoy them if they haven't dismissed the activate notice
+			echo '<div data-dismissible="activate-notice-forever" id="message" class="notice notice-warning is-dismissible"><p>' . __( '<b>' . 'OpenStax Import:' . '</b>' . ' Please activate WP QuickLaTeX for multiline equations and svg image export support. ' ) . '</p></div>';
+		} );
+		// ask book admin to activate it since the Network Admin has made it available to them
+		add_action( 'admin_notices', function () {
```

Comment:
there needs to be a condition here to check that they haven't already activated it at the book level, such as `get_option( 'active_plugins' );` for the book and `get_site_option( 'active_sitewide_plugins' );` for the network. I believe `get_plugins()` only is concerned with the existence of files in the directory, and not whether the user has activated it.